llvm/test/Transforms/LoopVectorize/vplan-printing-before-execute.ll

```diff
@@ -8,7 +8,7 @@ target datalayout = "e-m:o-i64:64-i128:128-n32:64-S128"
 ; VF/IC combination.
 define void @test_tc_less_than_16(ptr %A, i64 %N) {
 ; CHECK:      LV: Scalarizing:  %cmp =
-; CHECK-NEXT: VPlan 'Initial VPlan for VF={8},UF>=1' {
+; CHECK:      VPlan 'Initial VPlan for VF={8},UF>=1' {
```

Comment:
Should the additional line that broke the -NEXT be checked too?

Reply:
the purpose of the test is to check vplan's representation